beard.js

```diff
@@ -115,18 +115,12 @@ const parse = {
       _context.locals.pop();
     `;
   },
-  for: (_, key, value, object) => {
-    if (!value) {
-      value = key;
-      key = `_iterator_${uniqueIterator(value)}`;
-    }
+  for: (_, value, key, object) => {
+    if (!key) key = `_iterator_${uniqueIterator(value)}`;
```

Comment:
@joeosburn so did this used to be the old way because we didn't know if the first one was the key or the value?

Reply:
@shanebo Correct